src/ssr-template-helper.js

```diff
@@ -87,10 +84,7 @@ export class SsrTemplateHelper {
     if (!opt_templates) {
       const template = this.templates_.maybeFindTemplate(element);
       if (template) {
-        // The document fragment can't be used in the message channel API thus
-        // serializeToString for a string representation of the dom tree.
-        mustacheTemplate = this.xmls_.serializeToString(
-            this.templates_.findTemplate(element));
+        mustacheTemplate = template./*REVIEW*/innerHTML;
```

Comment:
Eh, we can probably make this API clearer. /cc @alabiaga 

```js
fetchAndRenderTemplate(element, request, successTemplate, opt_errorTemplate = null, opt_attributes = {})
```

No action needed. :)

Reply:
It's ok, I'd rather not name the params by its callers.

Reply:
Sure done.